gridstatus/miso.py

```diff
@@ -365,20 +365,27 @@ def get_lmp(
     ):
         """
         Supported Markets:
-            - ``REAL_TIME_5_MIN`` - (FiveMinLMP)
-            - ``DAY_AHEAD_HOURLY`` - (DayAheadExPostLMP)
-            - ``REAL_TIME_HOURLY_FINAL`` - (RealTimeFinalLMP)
-            - ``REAL_TIME_HOURLY_PRELIM`` - (RealTimePrelimLMP) Only 4 days of data
-                available, with the most recent being yesterday.
+            - ``REAL_TIME_5_MIN`` - (Prelim ExPost 5 Minute)
+            - ``DAY_AHEAD_HOURLY`` - (ExPost Day Ahead Hourly)
+            - ``REAL_TIME_HOURLY_FINAL`` - (Final ExPost Real Time Hourly)
+            - ``REAL_TIME_HOURLY_PRELIM`` - (Prelim ExPost Real Time Hourly)
+                Only 4 days of data available, with the most recent being yesterday.
         """
         if market == Markets.REAL_TIME_5_MIN:
             latest_url = "https://api.misoenergy.org/MISORTWDBIReporter/Reporter.asmx?messageType=currentinterval&returnType=csv"  # noqa
             today_url = "https://api.misoenergy.org/MISORTWDBIReporter/Reporter.asmx?messageType=rollingmarketday&returnType=csv"  # noqa
+            yesterday_url = "https://api.misoenergy.org/MISORTWDBIReporter/Reporter.asmx?messageType=previousmarketday&returnType=csv"  # noqa
 
             if date == "latest":
                 url = latest_url
             elif utils.is_today(date, tz=self.default_timezone):
                 url = today_url
+            elif utils.is_yesterday(date, tz=self.default_timezone):
```

Comment:
I don't think we should support the actual string "yesterday" because we don't do that with any other methods. Instead, we should require the caller to use yesterday's date.

Reply:
okay, that's fine